canonical-websites/discourse-docs.snapcraft.io
# Display old cache data on error

**Based on #34 - merge that first**

If the discourse API is down, this will return stale data from the cache instead of erroring.

Fixes #27

QA
--

First, make the cache time super-short by editing `webapp/models.py` and changing line 14 to:

``` python3
DEFAULT_SESSION = CachedSession(expire_after=1, old_data_on_error=True)
```

Now `./run`, go to http://0.0.0.0:8029/t/documentation-outline/3781. Then turn off your internet, and reload the page. It will take a while, but the page should still load successfully.
